client/layout/guided-tours/docs/TUTORIAL.md

```diff
@@ -0,0 +1,80 @@
+# Tutorial: Building a Simple Tour
+
+In this tutorial, we'll create a brief tour that shows a user how the site preview works. 
+
+It's not a comprehensive tutorial, but walks you through many of the different tasks that you'll encounter when developing a tour using Guided Tours. 
+
+For more complete technical information, please see the existing tours (linked to [in the README.md](../README.md)), the [API documentation](./docs/API.md), and the [architecture documentation](./ARCHITECTURE.md). 
+
+## What Do We Want To Achieve? 
+
+Before we think about what steps we want our tour to have, we should take a step back and think about what goals we want to achieve with our tour. 
+
+In this case, we might want to make sure new users understand where the site preview is and how it works. 
+
+In addition, it will be useful to have thought about how we want to measure reaching this goal. Here, we would hope that with our tour active, we would soon see an increase the usage of the site preview by new users. 
+
+## Decide for Tour Steps
+
+With that in mind, we can come up with the following steps:
+
+1. Ask the user whether they want to learn how to preview their site. 
+2. Point to the site preview and tell the user to click their site's title so that the preview opens. 
+3. When the preview is showing, explain that this is the site preview. 
+4. Show the user how to exit the preview and wait for them to do it. 
+5. Finish the tour. 
+
+We could have included a step — or even multiple steps — to explain the different device sizes, but it's advisable to start off with tours that are as short as possible to see how they perform. We can always come back and add additional steps. 
```

Comment:
Somewhere in this section, whether before or after the list, I'd make it clear that we're intentionally adding a lot of point for the sake of illustration, and that a better UX is generally one that minimizes interaction redundancy. What I mean by that is that, with most tours, developers should prefer an initial step that is ready for action: instead of asking for the user's time, directly point to the element(s) of interest; instead of adding a useless Next element, prefer a Continue step ready to react to the user's trigger (which is a confirmation of their interest to take the tour).